c_glib/test/test-decimal.rb

```diff
@@ -36,4 +36,14 @@ def test_abs
     decimal.abs
     assert_equal(absolute_value, decimal.to_s)
   end
+
+  def test_negate
+    absolute_value = "23049223942343532412"
```

Comment:
How about `positive_value` instead of `absolute_value`?
Because it seems that the value is treated as "positive" value not "absolute" in this context.

Reply:
Oh, that makes sense. I'll rename it.